releasenotes/notes/azure-app-services-extension-trace-agent-fix-c1238af92f2f015b.yaml

```diff
@@ -0,0 +1,6 @@
+---
+fixes:
+  - |
+    Restores the ability to run the trace-agent in the Azure App Services
```

Comment:
Nit: Rather than saying `run the trace-agent` I'd say `update the trace-agent`. We've always been able to run it but we were unable to update from a pinned version.